src/components/home/Home.jsx

```diff
@@ -0,0 +1,122 @@
+import Button from "../button/Button";
+import ServicesList from "../servicesList/ServicesList";
+
+import icon from "../../images/Polygon.svg";
+import icon1 from "../../images/icon1.svg";
+import icon2 from "../../images/icon2.svg";
+import icon3 from "../../images/icon3.svg";
+import arrow from "../../images/Arrow-right.svg";
+import arrowGrey from "../../images/Arrow-grey.svg";
+import img1_desktop from "../../images/picture1_desktop.png";
+import img2_desktop from "../../images/picture2_desktop.png";
+import img3_desktop from "../../images/picture3_desktop.png";
+import homeDesktop from "../../images/home_img_desktop.png";
```

Comment:
import {icon, icon1, icon2, icon3} from "../../images";

but before this you should export it from images